content/applications/finance/accounting/customer_invoices/terms_conditions.rst

```diff
@@ -2,7 +2,10 @@
 Default terms and conditions (T&C)
 ==================================
 
-Add default terms and conditions at the bottom of all customer invoices, sales orders, and
+Specifying terms and conditions is essential to establish important contractual points, such as
+payment terms, limitations of liability, and delivery terms between customers and sellers.
```

Comment:
Actually, with odoo, payment terms are a separate thing... but you could mention the penalty in case of late payment. ^^

I suggest writing examples that are more sales-oriented: return and refunds, warranty, after-sale services...